src/lib/operators/abstract_operator.cpp

```diff
@@ -60,22 +60,7 @@ void AbstractOperator::execute() {
 }
 
 // returns the result of the operator
-std::shared_ptr<const Table> AbstractOperator::get_output() const {
-  DebugAssert(
-      [&]() {
-        if (!_output) return true;
-        if (_output->chunk_count() <= ChunkID{1}) return true;
-        for (auto chunk_id = ChunkID{0}; chunk_id < _output->chunk_count(); ++chunk_id) {
-          if (_output->get_chunk(chunk_id)->size() < 1) return true;
-        }
-        return true;
-      }(),
-      "Empty chunk returned from operator " + description());
-
-  DebugAssert(!_output || _output->column_count() > 0, "Operator " + description() + " did not output any columns");
```

Comment:
The first `DebugAssert` is always true and can be removed.
But the second `DebugAssert` could be false and should also be removed?

Reply:
I removed these DebugAsserts because a) they were showing up in Debug build perf and b) because in #1631 @mrks and I seemed to agree that a Table without Columns or Rows isn't actually an invalid state. 

To keep this PR on-topic (i.e. release build perf) I put those two DebugAsserts back in for now.

Reply:
If it makes sense, keep it in. I was just surprised.

Reply:
Removed them again to be able to import a truly empty CSV